site/docs/relations/logical_relations.md

```diff
@@ -268,14 +268,23 @@ The set operation encompasses several set-level operations that support combinin
-| Intersection (Primary)  | Returns all records from the primary input that match at least one record from *any* secondary inputs.        | If a field is nullable in the primary input and in any of the secondary inputs, it is nullable in the output.
-| Intersection (Multiset) | Returns all records from the primary input that match at least one record from *all* secondary inputs.        | If a field is required in any of the inputs, it is required in the output.
-| Union Distinct          | Returns all the records from each set, removing any rows that are duplicated (within or across sets).         | If a field is nullable in any of the inputs, it is nullable in the output.
-| Union All               | Returns all records from each set, allowing duplicates.                                                       | If a field is nullable in any of the inputs, it is nullable in the output. |
+For some set operations, whether a specific record is included in the output and if it appears more than once depends on the number of times it occurs across all inputs. In the following table, treat:
+* m: the number of time a records occurs in the primary input
+* n1: the number of times a record occurs in the 1st secondary input
+* n2: the number of times a record occurs in the 2nd secondary input
+* ...
+* n: the number of times a record occurs in the nth secondary input
+
+| Operation                  | Description                                                                                                                                                                                                                                                   | Examples                                                                                                                                                         | Output Nullability
+|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------| -----------------------------
+| Minus (Primary)            | Returns all records from the primary input excluding any matching rows from secondary inputs, removing duplicate rows within or across sets.<br/>Each value is treated as a unique member of the set, so duplicates in the first set don’t affect the result. | MINUS<br/>&nbsp;&nbsp;m:  {1, 2, 2, 3, 3, 3, 4}<br/>&nbsp;&nbsp;n1: {1, 2}<br/>&nbsp;&nbsp;n2: {3}<br/>YIELDS<br/>{4}                                            | The same as the primary input.                                                                                                                                                                     
```

Comment:
Minor nit: I'm not entirely sure if "within or across" is adding value here.  Maybe just leave it at "removing duplicate rows".